usbsdmux/__main__.py

```diff
@@ -64,17 +64,38 @@ def main():
     ctl = UsbSdMux(args.sg)
     mode = args.mode.lower()
 
-    if mode == "off":
-        ctl.mode_disconnect()
-
-    elif mode in ("dut", "client"):
-        ctl.mode_DUT()
-
-    elif mode == "host":
-        ctl.mode_host()
-
-    elif mode == "get":
-        print(ctl.get_mode())
+    try:
+        if mode == "off":
+            ctl.mode_disconnect()
+
+        elif mode in ("dut", "client"):
+            ctl.mode_DUT()
+
+        elif mode == "host":
+            ctl.mode_host()
+
+        elif mode == "get":
+            print(ctl.get_mode())
+
+    except FileNotFoundError as fnfe:
+        print(fnfe, file=sys.stderr)
+        sys.exit(fnfe.errno)
```

Comment:
errno is unusual as an exit code. If you don't have anything else defined, just use sys.exit(1).

Reply:
It is fine to just return sys.exit(1). I will change it this way.